src/cryptoadvance/specter/node.py

```diff
@@ -142,16 +142,20 @@ def _get_rpc(self):
         else:
             rpc = None
         if self.autodetect:
-            if self.port:
-                rpc_conf_arr = autodetect_rpc_confs(
-                    self.node_type,
-                    datadir=os.path.expanduser(self.datadir),
-                    port=self.port,
-                )
-            else:
-                rpc_conf_arr = autodetect_rpc_confs(
-                    self.node_type, datadir=os.path.expanduser(self.datadir)
-                )
+            try:
+                if self.port:
+                    # autodetect_rpc_confs is trying a RPC call
+                    rpc_conf_arr = autodetect_rpc_confs(
+                        self.node_type,
+                        datadir=os.path.expanduser(self.datadir),
+                        port=self.port,
+                    )
+                else:
+                    rpc_conf_arr = autodetect_rpc_confs(
+                        self.node_type, datadir=os.path.expanduser(self.datadir)
+                    )
+            except BrokenCoreConnectionException:
+                return None
```

Comment:
Maybe it's worth mentioning the Exception-handling practices in the docstring. Maybe something like:
"Shouldn't throw any exception. Returns None if rpc is not working"
Something similiar would be good for other rpc-related functions/methods as well.

Reply:
Will clarify that.